nginx/setup.sh

```diff
@@ -1,20 +1,65 @@
+#!/bin/bash
+
+# Setup Nginx for local developement
+#
+# Copies Nginx configuration and SSL certificates to Nginx home (usually /usr/local/etc/nginx)
+# Make sure you have valid AWS credentials and then run with sudo setup.sh <profile name>
+# Remember to add nginx/hosts to your /etc/hosts
+
+S3_BUCKET="s3://identity-local-ssl/"
```

Comment:
a bucket just to store ssl certs sounds like an overkill. Can we store them in an already exisiting bucket in frontend s3?